ci/cfg/ci_ibex_fi_vcc_dummy.yaml

```diff
@@ -12,6 +12,13 @@ target:
 fisetup:
   fi_gear: "dummy"
   fi_type: "voltage_glitch"
+  # Parameter generation.
+  parameter_generation: "random"
+  # Number of iterations for the parameter sweep.
```

Comment:
Nit: `# Number of randomized iterations for the parameter sweep.` IIUC, this tells us how many randomized  parameters configs are drawn for each deterministic parameter config. This is nice.

Reply:
Changed the comment.